File: api/Params/Orders/OrderFwdParams.cs
namespace api.Params.Orders
{
    public class OrderFwdParams: PaginationParams
    {
        public int OrderId { get; set; }
        public int OrderNo { get; set; }
        public int OrderItemId { get; set; }
        public int ProfessionId { get; set; }
        public DateTime? FwdDateFrom { get; set; }
        public DateTime FwdDateUpto { get; set; } 
        public int CustomerId { get; set; }
    }
}

File: client/src/app/_models/params/orders/orderFwdParams.ts
export class OrderFwdParams
{
    orderNo=0;
    orderId=0;
    fwdDateFrom? = new Date();
    fwdDateUpto? = new Date();
    customerId: number = 0;
    professionId = 0;
    
    sort = "orderno";
    pageNumber = 1;
    pageSize = 10;
    search: string='';
}